src/components/TitlePicker.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import * as React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import {Picker} from '@react-native-picker/picker';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    alignItems: 'center',
+  },
+  label: {
+    flexBasis: '25%',
+    fontSize: 16,
+    marginVertical: 3,
+  },
+  value: {
+    flexBasis: '75%',
+    fontSize: 16,
+    marginVertical: 3,
+  },
+  pickerItem: {
+    fontSize: 16,
+  },
+});
+
+const TitlePicker = ({
+  label,
+  value,
+  onValueChange,
+}: {
+  label: string,
+  value: ?string,
+  onValueChange: ?(itemValue: string | number, itemIndex: number) => mixed,
```

Comment:
It's minor that `itemValue` => should be `value`, `itemIndex` => should be `index`. You don't need to change this, just remember

Reply:
Thank you for your feedback. The names were taken from the Picker component itself, but I think your argument makes much more sense. Hence, I have changed the parameters name accordingly.